LICENSE-binary

```diff
@@ -269,16 +269,19 @@ org.scala-lang:scala-reflect
 org.slf4j:jcl-over-slf4j
 org.yaml:snakeyaml
 
-COMMON DEVELOPMENT AND DISTRIBUTION LICENSE (CDDL) Version 1.0
+This project bundles the following dependencies under the CDDL 1.0 license.
+See license/LICENSE-CDDL.txt files for details.
 ------------
 javax.servlet:javax.servlet-api
 
-MIT License
+The MIT License (MIT)
+See license/LICENSE-MIT.txt files for details.
```

Comment:
As the slf4j license has the copyright , we cannot replace it with the MIT License template.

Reply:
Got it, we will add it back.